This test should trigger 'window.onerror' multiple times, and successfully handle the errors.

On success, you will see a series of "PASS" messages, followed by "TEST COMPLETE".


window.onerror: "Uncaught Error: Inline exception." at onerror-test.js (Line: 6, Column: 40)
Returning 'true': the error should not be reported in the console as an unhandled exception.
window.onerror: "Uncaught Error: exception in onload" at onerror-test.js (Line: 6, Column: 40)
Returning 'true': the error should not be reported in the console as an unhandled exception.
window.onerror: "Uncaught Error: exception in setTimeout" at onerror-test.js (Line: 6, Column: 40)
PASS successfullyParsed is true

TEST COMPLETE

